astropy/modeling/functional_models.py

```diff
@@ -91,6 +91,32 @@ class Gaussian1D(Fittable1DModel):
     amplitude = Parameter(default=1)
     mean = Parameter(default=0)
     stddev = Parameter(default=1)
+    _bbox = 'auto'
+
+    @property
+    def bbox(self):
+        '''
+        Returns the bounding box limits (x_low, x_high)
+        Default is (-5 * x_stddev, 5 * x_stddev)
+
+        model.bbox = 'auto' will reset the default.
+        '''
+        if self._bbox == 'auto':
```

Comment:
I need to think more on this too.  For `Model.inverse`, some models have a built-in analytic inverse.  However, for *all* models it is possible to set a custom inverse on a given instance, like `model.inverse = some_model`.  This is useful for cases, for example, where there is an inverse to a specific model given by observation, or some external calculation.  However, you can reset the model's inverse to its default by setting `model.inverse = None`.  This is fairly common practice but it's also counter-intuitive that it means "reset to default" rather than "model has no inverse".

I was going to say for bbox you should use `None` to reset to the default, but in this case you use `None` to disable the bounding box altogether.  

There's no right answer here, I just wonder what's best.  Another option would be to always have the bounding box (if it is defined for a model), and just enable/disable it in the `__call__` method when evaluating a model (ex. `model(x, bounded=False)` or `model(x, bounded=True)`).  If one truly wanted the bounding box itself to be effectively unbounded they could set `model.bbox = (-inf, inf)` or something like that.